e2e/tests/manageDocuments_test.js

```diff
@@ -329,9 +334,102 @@ Scenario('Solicitor with access uploads documents', async ({I, manageDocumentsEv
   caseViewPage.selectTab(caseViewPage.tabs.correspondence);
 
   assertCorrespondence(I, 'solicitor', 1, 'Email with evidence attached', 'Case evidence included');
+});
+
+Scenario('LA upload confidential and non confidential court bundle documents for a hearing', async ({I, caseViewPage, manageHearingsEventPage, manageDocumentsLAEventPage}) => {
+  await setupScenario(I);
+  await setupHearing(I, caseViewPage, manageHearingsEventPage);
+
+  // Update a confidential and a non-confidential document for the court bundle
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await uploadDocumentsToCourtBundleEvent(I, caseViewPage, manageDocumentsLAEventPage);
+
+  // Assert local authority can see both confidential and non-confidential documents on Court bundle tab
+  caseViewPage.selectTab(caseViewPage.tabs.courtBundle);
+  assertCourtBundleTabForHMCTSAndLA(I);
+
+  // Assert HMCTS users can see both confidential and non-confidential documents on Court bundle tab
+  await I.navigateToCaseDetailsAs(config.hmctsAdminUser, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.courtBundle);
+  assertCourtBundleTabForHMCTSAndLA(I);
+
+  // Assert solicitor can see the non-confidential document only on Court bundle tab
+  await I.navigateToCaseDetailsAs(config.privateSolicitorOne, caseId);
+  caseViewPage.selectTab(caseViewPage.tabs.courtBundle);
+  assertCourtBundleTabForSolicitor(I);
+
+  // Remove all documents from the court bundle
+  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await removeDocumentsFromCourtBundleEvent(I, caseViewPage, manageDocumentsLAEventPage);
 
+  // Assert no more documents on Court bundle tab
+  caseViewPage.selectTab(caseViewPage.tabs.courtBundle);
+  I.dontSeeTagInTab(['Court bundle 1', 'Documents 1']);
```

Comment:
I don't think this is required, although it would be very nice to have the e2e tests are the flakiest part of our pipeline so the less we are doing the better tbh

Reply:
OK I removed that.